.github/workflows/php-test.yml

```diff
@@ -58,13 +58,22 @@ jobs:
     steps:
       - uses: styfle/cancel-workflow-action@0.11.0
       - uses: actions/checkout@v3
+      - uses: shivammathur/setup-php@v2
+        with:
+          php-version: ${{ matrix.php }}
```

Comment:
Oh, I guess this isn't needed because it's running unit tests in the container.

Reply:
But it would allow `composer install` to work properly without having to run it in the container?